python/artm/__init__.py

```diff
@@ -0,0 +1,18 @@
+from artm_model import (
```

Comment:
- Example on StackOverflow: [why would I put python code in __init__.py files](http://stackoverflow.com/questions/5831148/why-would-i-put-python-code-in-init-py-files)
- Look at `__init__.py` of any `sklearn.<module>`, e.g. [sklearn.linear_model](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/linear_model/__init__.py)
- Look at any `__init__.py` in `gensim`, e.g. [gensim.models.__init__](https://github.com/piskvorky/gensim/blob/develop/gensim/models/__init__.py)

The general pattern consist in that you incapsulate all your auxiliary classes, functions, variables within the package in meaningfully separated py-modules. To make an interface for the user of your library, you add public classes and functions to the namespace of the package, this is done by imports in `__init__`.